rlpyt/replays/sequence/uniform.py

```diff
@@ -11,7 +11,7 @@ def set_batch_T(self, batch_T):
         self.batch_T = batch_T  # Can set dynamically.
 
     def sample_batch(self, batch_B, batch_T=None):
-        T_idxs, B_idxs = self.sample_idxs(batch_B, batch_T)
+        T_idxs, B_idxs, batch_T = self.sample_idxs(batch_B, batch_T)
```

Comment:
self.sample_idxs should return a batch_T in case of batch_T=None
Otherwise, self.extract_batchh will raise an error, because it cannot determine the length of sample sequence.